task.go

```diff
@@ -251,8 +251,19 @@ func (t *task) Resize(ctx context.Context, w, h uint32) error {
 	return err
 }
 
+const (
+	NoPivotRoot              = "io.runc.option.no-pivot-root"
```

Comment:
Ya, i'm not sure what we should do here.  Its a pretty complex problem of making a generic way for consumers to pass runtime specific options through the API.